prime-router/metadata/schemas/Experity/CUC-AL-covid-19.schema

```diff
@@ -0,0 +1,45 @@
+# Schema for Compass Urgent Care in Alabama
+# This schema is to be used to parse the standard lab report from experity
+# Created by Rick Hood
+# Last Update: 08/19/2021
+# 
+name: cuc-al-covid-19
+topic: covid-19
+description: Experity CSV lab report schema, Compass Urgent Care, AL
+extends: experity/experity-covid-19
+
+elements:
+
+    - name: sender_id
+      default: cuc-al
+
+    - name: processing_mode_code
+      default: P
```

Comment:
```suggestion
      default: T
```

Reply:
We need to run them in Training mode until you get signoff from the State that things are OK - this is a new step in our Onboarding, that has been requested by the receiver team.